web/templates/coinFlipper.html

```diff
@@ -3,7 +3,7 @@
 
 <!-- Specify title of tab -->
 {% block title %}
-    Coin Flipper
+    - Coin Flipper
```

Comment:
Why is this hyphen added?

Reply:
Because the root template title was changed to "Bored Games" thus the addition of the hyphen gives it the better looking "Bored Games - Coin Flipper"